### DISCUSSION TITLE
Duda

### DISCUSSION THREAD

**Discussion by bello79** [Launch-X-Latam/MisionFrontEnd#239] [General]
Buenas Tardes..! Pregunto, por las dudas, alguien sabes o hay un tutorial de como sincronizar mi GitHub con mi VS? Probe con la llave SSH del tutorial que esta en GitHUb pero no sube, en el VS me aparece como si que sube pero en mi GitHub nada. Mas que nada es por si hay algun otro tutorial como para ver si me funciona (o lograr identificar que estoy haciendo mal). Igualmente estoy subiendo de forma manual nomas los archivos al GitHub por ahora almenos.

**Comment by Sebastian-Rob-Cam** [comment]
Hola te refieres a sincronizar tu cuenta de GitHub con VS para guardar tus configuraciones? o para utilizar la consola de VS para subuir las actualizaciones de tu codigo a tu repositorio en GitHub?

**Comment by EsthebanDiego** [comment]
yo use este [Tutorial](https://www.youtube.com/watch?v=Fw4wxIFCpP8&t=3740s&ab_channel=TheFullstackDevs) para ver como subir los archivos desde VS code
